packages/contracts/src/DatabaseResolver.sol

```diff
@@ -8,19 +8,14 @@ import {OffchainLookup} from
     "@ens-contracts/dnsregistrar/OffchainDNSResolver.sol";
 import {IExtendedResolver} from
     "@ens-contracts/resolvers/profiles/IExtendedResolver.sol";
-import {IAddrResolver} from
-    "@ens-contracts/resolvers/profiles/IAddrResolver.sol";
-import {INameResolver} from
-    "@ens-contracts/resolvers/profiles/INameResolver.sol";
-import {IABIResolver} from "@ens-contracts/resolvers/profiles/IABIResolver.sol";
-import {IPubkeyResolver} from
-    "@ens-contracts/resolvers/profiles/IPubkeyResolver.sol";
-import {ITextResolver} from
-    "@ens-contracts/resolvers/profiles/ITextResolver.sol";
-import {IContentHashResolver} from
-    "@ens-contracts/resolvers/profiles/IContentHashResolver.sol";
-import {IAddressResolver} from
-    "@ens-contracts/resolvers/profiles/IAddressResolver.sol";
+import {AddrResolver} from "@ens-contracts/resolvers/profiles/AddrResolver.sol";
```

Comment:
If everything is overwritten, why use the implementation rather than the interface?

Reply:
because the interface only has the read method so it wasn't guaranteed that the DBResolver was overriding the right writing methods.

![image](https://github.com/user-attachments/assets/55a90fa9-01bd-43e2-8a21-05da207bee92)